Web/starter_project_barbies/types/blog.ts

```diff
@@ -0,0 +1,16 @@
+// Each blog has these properties
+export type Blog = {
+    userID : string;
+    blogID: string;
+    title: string;
+    content: string;
+    tags: string[];
+    date: string;
+    author: {
```

Comment:
for now this is fine but when data is loaded this should be modified where userId is either in the autor part, or you only have userId and you have to load the author separately (based on how the backend would work)

but nice for now